src/app/game/settings/account/change/email/page.tsx

```diff
@@ -0,0 +1,72 @@
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+import NavigationFooter from "@/features/game/NavigationFooter";
+import React from "react";
+import { ChevronLeft } from "lucide-react";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+export default function GameSettingsAccountDelete() {
+  const pageTitle = "設定";
+  const pageSubTitle = "メールアドレス変更";
+
+  return (
+    <div className="relative h-full">
+      <h1 className="fixed inset-x-0 top-0 bg-[#0094f4] p-4 pt-7 text-center text-3xl text-white">
+        {pageTitle}
+      </h1>
+
+      <div className="pb-16 pt-20">
+        <div className="relative mx-6 flex h-[80px] items-center justify-around">
+          <div className="absolute left-0">
+            <Link href="/game/settings" className="p-0">
+              <ChevronLeft color="#000000" size={50} />
+            </Link>
+          </div>
+          <h1 className="text-2xl">{pageSubTitle}</h1>
+        </div>
+      </div>
+
+      <div className="mx-auto w-4/5">
+        <p className="mb-2 text-xl">確認のため、<span className="font-bold">パスワード</span>を入力してください。</p>
```

Comment:
こちらも同行の修正依頼になります！
![image](https://github.com/user-attachments/assets/06b770ce-9323-49e4-9cb4-f51f8bb0887e)
Figmaで確認した所、こちらの文字の大きさは24pxとなっていました。TailwindCSSでは`text-2xl`で24pxが表現できます。
他のページも24pxで合わせているため、修正依頼を出させてもらいます。
```suggestion
        <p className="mb-2 text-2xl">確認のため、<span className="font-bold">パスワード</span>を入力してください。</p>
```